packages/containers-shared/eslint.config.mjs

```diff
@@ -0,0 +1,4 @@
+import sharedConfig from "@cloudflare/eslint-config-shared";
+import { defineConfig, globalIgnores } from "eslint/config";
+
+export default defineConfig(globalIgnores(["src/client/**"]), sharedConfig);
```

Comment:
`src/client` is a generated API client. Previously it wasn't ignored, which was OK in CI because containers-shared didn't use `max-warnings=0` when running eslint